decentraland/helpers/storage.js

```diff
@@ -132,9 +133,13 @@ const _storeData = async (key, value, securely) => {
 const _clearData = async key => {
   try {
     await AsyncStorage.removeItem(key);
+  } catch (error) {
+    logInfo(`Unable to delete data (key = ${key}) from storage.`);
```

Comment:
`INFO` is a lower severity log than `WARN`. I would argue this is something that warrants a `WARN` level.

Reply:
(Same for the other error for the secure storage)

Reply:
Not important for this PR really, but mostly to set precedent for future code

Reply:
Got it

Reply:
I've made that way because if an exception is thrown for example if the key doesn't exist it'll log a warn on every clear attempt.

Reply:
Oh, hmmm. That's an interesting point. So there is a pretty good argument for it being INFO, or splitting out based on whether that's the type of exception or if it's a different type of exception. But for now I'll merge this with how you have it, and we can ponder this going forward without there being an issue for it.